spec/rails_spec.rb

```diff
@@ -28,7 +28,7 @@ def sh(command, options={})
           ENV.delete("RAILS_ENV")
           ENV.delete("RACK_ENV")
 
-          sh "bundle config --local --path vendor/bundle"
+          sh "bundle config --local path vendor/bundle"
```

Comment:
Are you sure? Did you try running it?

```
$ bundle config --local --path vendor/bundle
You are replacing the current local value of --path, which is currently nil
$ cat .bundle/config 
---
BUNDLE_--PATH: "vendor/bundle"
```

That sounds incorrect to me. As I explained in the commit message, I was getting a lot of warnings coming from system gems because this setting was not being applied.

Reply:
ah yes ... this is not `bundle install` ... idk how that ever worked :D